src/bi-weapons.yaml

```diff
@@ -95,8 +95,6 @@ ChainGun.Yak:
 		Damage: 3500
 		Versus:
 			Steel: 105
-	Projectile: Bullet
-		Speed: 682
```

Comment:
I can't find this change in the current release (you said you checked against bleed, but this project has to work with a release/play test).

Reply:
It's there - all good. Sorry - I got confused because GitHub's automated context told me this change related to `ChainGun.Yak`.